src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -167,5 +194,10 @@ public OtlpExporterOptions()
         /// </list>
         /// </remarks>
         public Func<HttpClient> HttpClientFactory { get; set; }
+
+        /// <summary>
+        /// Gets a value indicating whether <see cref="Endpoint" /> was programmatically modified.
```

Comment:
how about being more explicit?

```suggestion
        /// Gets a value indicating whether <see cref="Endpoint" /> was modified via its setter.
```